src/providers/ad/ad_gpo.c

```diff
@@ -3456,7 +3456,7 @@ ad_gpo_get_som_attrs_done(struct tevent_req *subreq)
     }
 
     if ((ret == ENOENT) || (el->num_values == 0)) {
-        DEBUG(SSSDBG_OP_FAILURE, "no attrs found for SOM; try next SOM\n");
+        DEBUG(SSSDBG_MINOR_FAILURE, "no attrs found for SOM; try next SOM\n");
```

Comment:
This message is the same as above and I think this is wrong.
Looking below I guess it should be something like "AD_AT_GPLINK attr not found or has no value"?

Reply:
Ack.